plot.go

```diff
@@ -80,18 +80,25 @@ type DataRanger interface {
 	DataRange() (xmin, xmax, ymin, ymax float64)
 }
 
+type orientation bool
```

Comment:
This is unexported, undocumented and has no methods. Is it needed? The consts can be untyped boolean.